elpy.el

```diff
@@ -3292,13 +3293,161 @@ Returns a possible multi-line docstring."
             success error))
 
 (defun elpy-rpc-get-usages (&optional success error)
+  "Return the symbol under point usages as a list"
   (elpy-rpc "get_usages"
             (list buffer-file-name
                   (elpy-rpc--buffer-contents)
                   (- (point)
                      (point-min)))
             success error))
 
+(defun elpy-rpc-get-names (&optional success error)
+  "Return all names (possible candidates for jumping to definition ot others)."
+  (elpy-rpc "get_names"
+            (list buffer-file-name
+                  (elpy-rpc--buffer-contents)
+                  (- (point)
+                     (point-min)))
+            success error))
+
+;;;;;;;;;;;;;;;;
+;;; Xref backend
+
+;;;###autoload
+(defun elpy--xref-backend () 'elpy)
+
+(defun elpy-xref--set-backend ()
+  (setq-local xref-backend-functions 'elpy--xref-backend))
```

Comment:
Shouldn't this be a list? Is there a reason why we preclude other xref backends?